lib/modules/dosomething/dosomething_image/dosomething_image.module

```diff
@@ -41,12 +41,20 @@ function dosomething_image_get_themed_image($nid, $ratio, $style, $alt = '', $at
   else {
     // Get the path of the themed image.
     $path = dosomething_image_get_themed_image_url($nid, $ratio, $style);
+
+    if (!$path) {
+      return NULL;
```

Comment:
Would an empty string make more sense here, since it seems like it otherwise returns an HTML string?